tests/jballerina-unit-test/src/test/resources/test-src/query/simple-query-with-var-type.bal

```diff
@@ -195,14 +195,6 @@ function testFilteringNullElements() returns Person[] {
     return outputPersonList;
 }
 
-function testMapWithArity() returns boolean {
-    map<any> m = {a: "1A", b: "2B", c: "3C", d: "4D"};
-    var val = from var v in m
-                   where <string> v == "1A"
-                   select <string>v;
-    return val == ["1A"];
-}
```

Comment:
Can't we turn this into a positive test case if not available?
Also, did you add a negative test for this case?

Reply:
There are negative tests for map types in [query-expr-query-construct-type-negative.bal](https://github.com/ballerina-platform/ballerina-lang/pull/37547/files#diff-d872f8d1b5d70e08e4a1e2c3d798402ed49735fd594cf13144d2e0fa6fb4f8f3)

Reply:
updated